packages/wallets/src/wallets.ts

```diff
@@ -27,6 +29,13 @@ export interface Wallet {
 
 export const ICONS_URL = 'https://raw.githubusercontent.com/solana-labs/wallet-adapter/master/packages/wallets/icons';
 
+export const getBitpieWallet = (config?: Coin98WalletAdapterConfig): Wallet => ({
+    name: WalletName.Coin98,
+    url: 'https://coin98.com/',
```

Comment:
```suggestion
    url: 'https://coin98.com',
```

Just removing the trailing slash for consistency between the wallets.